tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivativeImpl.tpp

```diff
@@ -1008,25 +1017,49 @@ void test_impl(const Spectral::Quadrature quadrature,
         get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i] = 25.5;
         get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i + 1] = 37.5;
       } else if (quadrature == Spectral::Quadrature::Gauss) {
-        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i] = 25.5;
-        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i + 1] = 25.5;
+        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i] =
+            25.8660254037844375;
+        get(get<::Tags::dt<Var1>>(expected_dt_evolved_vars))[i + 1] =
+            38.598076211353316;
```

Comment:
There are a lot of magic numbers floating around in this test case (here and in diffs below).

It would help if the math being coded up (and presumably tested against) were written down someplace — is it? If not, can it be?

Reply:
I'll remind myself and add a comment, but this is basically a "I did DG 'by hand' and this is what we should get". It's more of a regression test. I didn't feel super enthusiastic about implementing the entire DG algorithm on a moving mesh in python. I want to test the basics get the right numbers, then test that the correct functions are called for moving mesh, etc. by effectively redoing the computation. I'll see what I can remind my self of and add a comment to the test explaining what's happening

Reply:
A comment (even if brief) would be very helpful, thanks! 👍